### ISSUE TITLE
Occupancy Class variable name inconsistent across programs.

### ISSUE THREAD

**Issue by plrvn** [nhrap-hazus/FAST#1]
Pre-process.py script uses OCC for Occupancy Class variable name and hazus.py script uses OccupancyClass. Can these be standardized to OccupancyClass across programs?

**Comment by dbausch2000** [comment]
Yes.  Thanks for pointing out.  Been thinking we would develop an ini or similar approach to provide consistent variable names and the opportunity for user to customize.

**Comment by ujvalak** [comment]
Hello Paul, 
Thank you for mentioning this. Both the pre-process and the main scripts show up red if the field name for any of the required field is not found in the chosen csv file. The  also accept the name by typing it in. For e.g. if the name in the csv file is "Occt" , the user can type it up and if it matches the box will turn green which means that the name is found and accepted.
We also intend to set up a XML file that will store the global properties and metadata for FAST in the future. 
Please see attached image that shows how the user can enter a field-name for the required fields that are may not exist in the accepted list.
![FieldEntry](https://user-images.githubusercontent.com/10346175/70755920-05027f80-1cf0-11ea-8595-6d9d0dcc9615.jpg)
